report_api/runreport.py

```diff
@@ -1,3 +1,5 @@
 #!.venv/bin/python
+import sys
+sys.path.append(sys.path[0])
```

Comment:
I added it specifically because it wasn't there, e.g. When running it from the regression directory (directly - not using run.sh because that creates another virtualenv which causes issues)